covasim/webapp/index.html

```diff
@@ -36,199 +36,222 @@
     </b-modal>
 
 
   </header>
 
   <div class="content">
 
-    <b-sidebar id="parameters" :visible="panel_open" no-header bg-variant="white" :width="panel_width+'vw'" shadow>
+    <b-sidebar id="parameters" :visible="panel_open" no-header class="sidebar" :width="panel_width+'vw'" shadow>
       <div class="px-3">
         <header class="d-flex justify-content-between">
-          <h4 class="my-2">Campaign Builder</h4>
-          <button class="close" @click="close_panel" aria-label="Close">
-            <span aria-hidden="true">&times;</span>
+          <h4 class="my-2">Simulation configuration</h4>
+          <button class="close" id="hidepanel" @click="close_panel" aria-label="Left">
+            <span aria-hidden="true">&#129092;</span>
           </button>
+          <b-tooltip target="hidepanel">
+            Hide configuration panel
+          </b-tooltip>
         </header>
+      </div>
 
-        <!-- Presets -->
-        <div class="form-row mb-3">
-          <div class="col">
-            <select :disabled="running" id="examples" v-model="reset_choice" @change="resetPars()" class="form-control" aria-label="Example Presets">
-              <option v-for="(item, index) in reset_options" :value="item">
-                {{ item }}
-              </option>
-            </select>
-            <b-tooltip target="examples">
-              Example parameter configurations
-            </b-tooltip>
-          </div>
-          <div class="col-auto">
-            <button id="reset_btn" :disabled="running" class="btn btn-primary blueish" type="button" @click="resetPars">
-              Reset
-            </button>
-            <b-tooltip target="reset_btn">
-              Reset parameters to default values
-            </b-tooltip>
-          </div>
-          <div class="col-auto">
-            <button id="download-parameters-button" :disabled="running" class="btn btn-primary blueish" type="button" @click="downloadPars">
-              <span class="ti-download"/>
-              <span class="sr-only"> Download parameters </span>
-            </button>
-            <b-tooltip target="download-parameters-button">
-              Download parameters
-            </b-tooltip>
-          </div>
-          <div class="col-auto">
-            <button id="upload-parameters-button" :disabled="running" class="btn btn-primary blueish" type="button" @click="uploadPars">
-              <span class="ti-upload"/>
-              <span class="sr-only"> Upload parameters </span>
-            </button>
-            <b-tooltip target="upload-parameters-button">
-              Upload parameters
-            </b-tooltip>
-          </div>
+      <div class="card mb-3">
+        <div class="card-header">
+          <button class="btn btn-block parheading" v-b-toggle.general>General options</button>
         </div>
+        <b-collapse visible id="general">
```

Comment:
All of the layout changes here (adding another collapse and  nesting the form rows) broke the sidebar maintaining the correct height on scroll after running the model. Please revert.

Reply:
I think we need to keep the collapsible "General" options, but have added it to the refactor: https://github.com/InstituteforDiseaseModeling/covasim/issues/180

Reply:
I don't think we need to collapse the general options. They're general because we decided in that meeting on the 7th that those were the most common options. Why would you hide them?

Reply:
If you are working with a lot of interventions, it can get a bit cramped otherwise. I was getting annoyed at not being able to collapse to save screen real estate